eip-0001.md

```diff
@@ -30,19 +33,22 @@ A new request to scan is initiated which registers said scan to be checked for a
 A predicate (function which returns a boolean value for a box) is required to register a scan.
 Predicates available are:
 
-* `CONTAINS(register, bytes)` returns true if certain register contains given bytes. If *register* is missed, R1 will be scanned
-* `EQUALS(register, bytes)`returns true if certain register contains only given bytes. If *register* is missed, R1 will be scanned
+* `CONTAINS(register, value)` returns true if certain register contains given value. If *register* argument is missed, R1 (script register) will be scanned
+* `EQUALS(register, value)`returns true if certain register contains only given value. If *register* argument is missed, R1 will be scanned
 * `OFTYPE(register, typeId)` - if certain register is of certain type
```

Comment:
What is typeId, is it Sigma type serialized bytes?

Reply:
Actually, not implemented, so removed